app/models/token.rb

```diff
@@ -22,20 +22,20 @@ def self.size_names
   end
 
   def name
-    read_attribute(:name).presence || tokenable.try(:name)
+    read_attribute(:name).presence || token_template.try(:name)
```

Comment:
Maybe not in this PR, but to confirm, is the plan to just always pull the name from `TokenTemplate` and drop `Token#name`?

Reply:
Yes, I think so